std/net/curl.d

```diff
@@ -525,6 +528,54 @@ if (isCurlConn!Conn)
     }
 }
 
+/** HTTP head
```

Comment:
Maybe the other symbols aren't ideal either, but I think a more descriptive summery should be used here.